nav2_bringup/bringup/launch/nav2_localization_launch.py

```diff
@@ -35,17 +34,14 @@ def generate_launch_description():
     autostart = LaunchConfiguration('autostart')
     params_file = LaunchConfiguration('params_file')
     use_lifecycle_mgr = LaunchConfiguration('use_lifecycle_mgr')
-    use_remappings = LaunchConfiguration('use_remappings')
 
     # Map fully qualified names to relative ones so the node's namespace can be prepended.
     # In case of the transforms (tf), currently, there doesn't seem to be a better alternative
     # https://github.com/ros/geometry2/issues/32
     # https://github.com/ros/robot_state_publisher/pull/30
     # TODO(orduno) Substitute with `PushNodeRemapping`
     #              https://github.com/ros2/launch_ros/issues/56
-    remappings = [((namespace, '/tf'), '/tf'),
-                  ((namespace, '/tf_static'), '/tf_static'),
-                  ('/tf', 'tf'),
+    remappings = [('/tf', 'tf'),
```

Comment:
Great! Not sure why I didn't do this simplification before. Sadly CI is broken to verify, but this should work. Remapping `/tf` to `tf` means that a node subscribing to `/tf` will effectively subscribe to `tf` (or `node_namespace/tf` and publishing to `/tf` will publish to `tf` (or `node_namespace/tf`).